analyzers/tests/SonarAnalyzer.UnitTest/LiveVariableAnalysis/RoslynLiveVariableAnalysisTest.LocalFunction.cs

```diff
@@ -275,15 +275,15 @@ public void LocalFunctionInvocation_CrossReference_WhenAnalyzingLocalFunctionIts
 int LocalFunction()
 {
     var variable = 42;
-    if (boolCondition)
+    if (boolParameter)
```

Comment:
review note: `CreateContextCS` puts this snippet inside a function that has `boolParameter` parameter